app/models/roles_user.rb

```diff
@@ -1,4 +1,26 @@
 class RolesUser < ApplicationRecord
   belongs_to :user
   belongs_to :role
+
+  after_create :log_role_addition
+  def log_role_addition
+    admin = User.current_user
+    note = "Change made by #{admin&.login}"
+    user = User.find_by(id: user_id)
```

Comment:
The model has `belongs_to :user` so you can skip the lookup here and simply refer to `user`.